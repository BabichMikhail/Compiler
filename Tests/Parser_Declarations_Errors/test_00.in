const a = 10;
var
	b : integer = 15;
	arr : array[A..B] of integer;