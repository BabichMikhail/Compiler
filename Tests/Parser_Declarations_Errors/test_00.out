Error: Const Expression expected. Line 4 Column 21
